msteams-platform/bots/build-a-bot.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Let's build a bot
+description: Learn to build a basic chatbot using Teams Toolkit, Bot Framework SDK. Additionally, learn to build an AI bot.
+ms.localizationpriority: high
+ms.date: 09/25/2024
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+# Let's build a bot
+
+Building a bot app involves various phases, from initial concept to final deployment. This article offers a detailed guide to help you get started.
+
+If you want to create your own bot, here's a list of tools and platforms to help you get started:
+
+:::image type="content" source="../assets/images/bots/m365-bot-decisions.png" alt-text="Infographics depicts a decision tree for building a bot.":::
+
+| # | Choose ... | From the following options ... |
+| --- | --- | --- |
+| 1. | Bot capabilities | - AI bot <br> - Non-AI bot: <br> &nbsp;&nbsp; • Conversational capability <br> &nbsp;&nbsp; • Notification capability <br> &nbsp;&nbsp; • Workflow capability <br> &nbsp;&nbsp; • Command capability |
```

Comment:
For AI bot, how about adding these further options:
- Teams AI bot
- Custom engine agent for Microsoft 365 Copilot

Reply:
@erikadoyle I've added the recommended bullets with some description.